clang/test/Driver/sycl-instrumentation.c

```diff
@@ -1,14 +1,17 @@
 /// Check that SPIR ITT instrumentation is disabled by default:
-// RUN: %clang -### %s 2>&1 \
+// RUN: %clangxx -fsycl -### %s 2>&1 \
 // RUN: | FileCheck -check-prefix=CHECK-DEFAULT %s
 // CHECK-DEFAULT-NOT: "-fsycl-instrument-device-code"
 
-/// Check if "fsycl_instrument_device_code" is passed to -cc1:
-// RUN: %clang -### -fsycl-instrument-device-code %s 2>&1 \
+/// Check if -fsycl-instrument-device-code is passed to device-side -cc1:
+// RUN: %clangxx -fsycl -### -fsycl-instrument-device-code %s 2>&1 \
 // RUN: | FileCheck -check-prefix=CHECK-ENABLED %s
-// CHECK-ENABLED: "-cc1"{{.*}} "-fsycl-instrument-device-code"
+// CHECK-ENABLED: "-cc1"{{.*}} "-fsycl-is-device"{{.*}} "-fsycl-instrument-device-code"
 
-/// Check if "fsycl_instrument_device_code" usage with a non-spirv target
+/// Check if -fsycl-instrument-device-code usage with a non-spirv target
 /// results in an error.
 // RUN: %clang -### -fsycl-instrument-device-code --target=x86 %s 2>&1
 // expected-error{{unsupported option '-fsycl-instrument-device-code' for target 'x86_64-unknown-linux-gnu'}}
+/// However, no error is expected for a non-SPIR host side
```

Comment:
So the negative test is unexpected now, correct?
nit: actually, it can be done for nvptx target, but I believe it's not required to be done in this patch, since short-middle-turn plan is to remove the option.

Reply:
The negative test is still required. Good point regarding the NVPTX target, I'll update the test accordingly.